apps/web-giddh/src/app/shared/header/components/account-add-new-details/account-add-new-details.component.ts

```diff
@@ -1235,16 +1239,19 @@ export class AccountAddNewDetailsComponent implements OnInit, OnChanges, AfterVi
      * @memberof AccountAddNewDetailsComponent
      */
     public onlyPhoneNumber(): void {
-        const input = document.getElementById('init-contact-add');
-        this.intl = new window['intlTelInput'](input, {
-            nationalMode: false,
-            utilsScript: MOBILE_NUMBER_UTIL_URL,
-            autoHideDialCode: false,
-            separateDialCode: false,
-            initialCountry: this.activeCompany?.countryV2?.alpha2CountryCode.toLowerCase(),
-            geoIpLookup: (success, failure) => {
-                let countryCode = this.activeCompany?.countryV2?.alpha2CountryCode.toLowerCase();
-                if (!countryCode) {
+        let input = document.getElementById('init-contact-add');
```

Comment:
Check if we can move this logic to common utility function

Reply:
Prateek sir we will move the common component to the shared module that will use in common and when we will do it for other modules.